### PR TITLE
New Command - SPDF ✅️

### PR DESCRIPTION
**Pull request by TOXIC-DEVIL**

### CODE REVIEW

**Review by phaticusthiccy — CHANGES_REQUESTED** [review]

File: language/EN.json [hunk 1]
```diff
@@ -424,6 +424,7 @@
         "webss": {
             "SS_DESC": "Takes a screenshot from the page in the given link.",
             "LİNK": "You must enter link!"
```
[the remark this answers is not in the document]
Comment (TOXIC-DEVIL) [reply language/EN.json]: Okay 🥰

**Review by phaticusthiccy — APPROVED** [review]

File: language/EN.json [hunk 2]
```diff
@@ -423,8 +423,8 @@
         },
         "webss": {
             "SS_DESC": "Takes a screenshot from the page in the given link.",
-            "LİNK": "You must enter link!"
-            "SPDF_DESC": "Converts a Site into PDF"
+            "LİNK": "You must enter link!",
+            "SPDF_DESC": "Converts a Site into PDF",
```
Comment (phaticusthiccy) [inline language/EN.json:427]: The end of the strings, you don't need to add **,**
`"SPDF_DESC": "Converts a Site into PDF"`

**Review by phaticusthiccy — CHANGES_REQUESTED** [review]

File: language/EN.json (hunk 2 above)
Comment (TOXIC-DEVIL) [reply language/EN.json:427]: This is okay na? What is the problem I have added , at the end na 💔😟

Comment (phaticusthiccy) [reply language/EN.json:427]: Check line 427. 
Corrupted Code: `"SPDF_DESC": "Converts a Site into PDF",`
Envisaged Code: `"SPDF_DESC": "Converts a Site into PDF"`

Comment (TOXIC-DEVIL) [reply language/EN.json]: Okay Now?! 😅

**Review by phaticusthiccy — APPROVED** [review]

**Comment by phaticusthiccy** [comment]
Suspended until new version of WhatsAsena 1.3.2

Reason: Eva Full Functionality Artificial Intelligence Features.

**Comment by TOXIC-DEVIL** [comment]
> Suspended until new version of WhatsAsena 1.3.2
> 
> Reason: Eva Full Functionality Artificial Intelligence Features.

Okay 💕

**Comment by TOXIC-DEVIL** [comment]
> This PR is stale because it has been open 5 days with no activity. Remove the stale label or comment or this will be closed in 2 days

Github 😁

**Comment by TOXIC-DEVIL** [comment]
> This PR is stale because it has been open 5 days with no activity. Remove the stale label or comment or this will be closed in 2 days

😖

**Comment by phaticusthiccy** [comment]
This Pull Request Will Add to [WhatsAsena Plugin Channel.](https://t.me/remasterplugin)
Projected Time: 0-1 Day

**Pull request closed without merging**